tools-local/unix/tool-helper.sh

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env bash
+
+# Provides helper functions.
+
+# Gets the repository root path.
+get_repo_root()
+{
+    scriptpath="$(cd "$(dirname "$0")"; pwd -P)"
+    repoRoot="$(cd "$scriptpath/../.."; pwd -P)"
+    echo "$repoRoot"
+}
+
+# Gets the path to Tools/downloads folder under repository root.
+# Exit 1 if unable to determine the path.
+get_repository_tools_downloads_folder()
+{
+    repoRoot="$(get_repo_root)"
+    toolsPath="$repoRoot/Tools/downloads"
+    
+    if [ -z "$toolsPath" ]; then
+        echo "Unable to determine repository tools path."
+        exit 1
+    fi
+
+    echo "$toolsPath"
+}
+
+# Gets the path to shell scripts in tools-local folder under the repository root.
+# Exit 1 if unable to determine the path.
+get_repository_shell_scripts_path()
+{
+    repoRoot="$(get_repo_root)"
+    shellScriptsPath="$repoRoot/tool-local/unix"
+
+    if [ -z "$shellScriptsPath" ]; then
+        echo "Unable to determine shell scripts path."
+        exit 1
+    fi
+
+    echo "$shellScriptsPath"    
+}
+
+# Eval .toolversions file.
+eval_tools()
+{
+    repoRoot="$(get_repo_root)"
+
+    # Dot source toolversions file.
+    . "$repoRoot/.toolversions"
+
+    eval "tools=\$$toolName"
```

Comment:
Hmm, I think `toolName` should be a parameter, and I'm not sure the `s` in `eval_tools` is accurate since it's just getting one.